### PR TITLE
WVD Scaling Tool: conditionally set PS exec policy to support CloudShell in deployment scripts

### PR DESCRIPTION
**Pull request by OxHobbs**
update WVD scaling tool deployment scripts to conditionally set the execution policy to account for running the script from cloud shell.

### CODE REVIEW

File: wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1 [hunk 1]
```diff
@@ -46,8 +46,10 @@ if (!$UseRDSAPI) {
 	$WebhookName += 'ARMBased'
 }
 
-# Set the ExecutionPolicy
-Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser -Force -Confirm:$false
+# Set the ExecutionPolicy if not being ran in CloudShell as this command fails in CloudShell
+if (-not ($env:POWERSHELL_DISTRIBUTION_CHANNEL -eq 'CloudShell') {
```
Comment (nakranimohit0) [inline wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1:50]: Can this be simplified to `if ($env:POWERSHELL_DISTRIBUTION_CHANNEL -ne 'CloudShell') {`

Comment (nakranimohit0) [inline wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1:50]: `CreateOrUpdateAzLogicApp.ps1` also needs to be updated with this change.

Comment (OxHobbs) [reply wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1]: Added the same change to both scripts

Comment (OxHobbs) [reply wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1]: good call - done

**Comment by nakranimohit0** [comment]
@OxHobbs Thank you for contributing !

**Pull request merged**